llvm/include/llvm/ADT/SmallSet.h

```diff
@@ -155,6 +150,22 @@ class SmallSet {
   using const_iterator = SmallSetIterator<T, N, C>;
 
   SmallSet() = default;
+  SmallSet(const SmallSet &) = default;
+  SmallSet(SmallSet &&) = default;
+
+  template <typename IterT> explicit SmallSet(IterT Begin, IterT End) {
```

Comment:
```suggestion
  template <typename IterT> SmallSet(IterT Begin, IterT End) {
```

Reply:
Thanks. I will address your comments in the stack of PRs that I will create instead of the current single PR.